Flow.Launcher/ViewModel/MainViewModel.cs

```diff
@@ -775,7 +833,7 @@ public async void Hide()
                 default:
                     throw new ArgumentException($"wrong LastQueryMode: <{_settings.LastQueryMode}>");
             }
-
+            _lastHistoryIndex = 1;
```

Comment:
why is this needed here and why 1?

Reply:
Resets the index when Flow is deactivated/hidden